OSDOCS#12433: Add disconnected support for OLM v1 (#86791)
openshift/openshift-docs: modules/oc-mirror-updating-restricted-cluster-manifests.adoc

Mirroring the {product-title} content using the oc-mirror OpenShift CLI (oc) plugin creates resources, which include `catalogSource-certified-operator-index.yaml` and `imageContentSourcePolicy.yaml`.

* The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry.
* The `CatalogSource` resource is used by {olmv0-first} to retrieve information about the available Operators in the mirror registry, which lets users discover and install Operators.
+
[NOTE]
====
{olmv1} uses the `ClusterCatalog` resource to retrieve information about the available cluster extensions in the mirror registry.

> The oc-mirror plugin v1 does not generate `ClusterCatalog` resources automatically, and they must be manually created. The oc-mirror plugin v2 does, however, generate `ClusterCatalog` resources automatically.

Reviewer: Passive/active
```suggestion
The oc-mirror plugin v1 does not generate `ClusterCatalog` resources automatically, and you must manually created. The oc-mirror plugin v2 does, however, generate `ClusterCatalog` resources automatically.
```

* The `ImageContentSourcePolicy` resource associates the mirror registry with the source registry and redirects image pull requests from the online registries to the mirror registry.
* The `CatalogSource` resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry, which lets users discover and install Operators.
+
[NOTE]
====
{olmv1-first} uses the `ClusterCatalog` resource to retrieve information about the available cluster extensions in the mirror registry.

> Currently, oc-mirror plugin v2 does not generate `ClusterCatalog` resources by default, and they must be manually created. For more information, see "Configuring your cluster to use {olmv1} in disconnected environments".

Reviewer 2: ```suggestion
Currently, oc-mirror plugin v1 does not generate `ClusterCatalog` resources by default, and they must be manually created. For more information, see "Configuring your cluster to use {olmv1} in disconnected environments".
```
